src/Overview/Overview.js

```diff
@@ -1,38 +1,121 @@
 import React from "react";
-import { StyleSheet, Text, View } from "react-native";
-import { Button } from "react-native-paper";
+import { StyleSheet, Text, View, TouchableOpacity } from "react-native";
 
-export default Overview = (props) => {
-  console.log(props.route.params.rooms);
-  // let rooms = props.map((room) => {
-  //   return (
-  //     // Some shit to do with room
-  //     <View>
-  //       <Text>LIVING ROOM</Text>
-  //     </View>
-  //   );
-  // });
-  const buttonPress = () => {
-    props.navigation.navigate("Room");
+export default Overview = ({ navigation, route }) => {
+  let kitchen = route.params.rooms[0];
+  let livingRoom = route.params.rooms[1];
+  let backyard = route.params.rooms[2];
+  let bedroom = route.params.rooms[3];
```

Comment:
This really does look great! Going to have to explain these lines to me though, not sure whats happening here.